app/views/organisations/_contacts.html.erb

```diff
@@ -0,0 +1,52 @@
+<% contacts.each do |contact| %>
+  <div class="organisation__contact-section <%= "organisation__border-top" if local_assigns[:border] %>">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: contact[:title],
+      padding: true,
+      heading_level: 3,
+      font_size: 19,
+    } %>
+
+    <div class="grid-row">
+      <% if contact[:post_addresses].any? %>
+        <address class="column-half organisations__margin-bottom organisation__address">
+          <% contact[:post_addresses].each do |post| %>
+            <%= post.html_safe %>
+          <% end %>
+        </address>
+      <% end %>
+
+      <% if contact[:email_addresses].any? || contact[:links].any? || contact[:phone_numbers].any?  %>
+        <div class="<%= contact[:post_addresses].any? ? "column-half" : "column-two-thirds" %>">
```

Comment:
Just wondering about the `column-two-thirds` here. I see what it does, is it worth the effort or could we use the full width in this circumstance?